.gitignore

```diff
@@ -21,3 +21,6 @@ npm-debug.log
 
 # Byebug command history file
 .byebug_history
+
+# Yarn error files
+yarn-error.log
```

Comment:
FYI: I usually ignore all `*.log` files.

Reply:
That would make sense